mx.sulong/mx_sulong.py

```diff
@@ -512,6 +520,22 @@ def runCompileTestCases(args=None):
     vmArgs, _ = truffle_extract_VM_args(args)
     return unittest(getCommonUnitTestOptions() + vmArgs + ['com.oracle.truffle.llvm.test.TestGCCSuiteCompile'])
 
+def runTestJRuby(args=None):
+    """tests that JRuby can use this version of Sulong to compile and run C extensions"""
+    rubyUrl = 'http://github.com/jruby/jruby.git'
+    rubyBranch = 'truffle-head'
+    suitesDir = os.path.abspath(join(_suite.dir, '..'))
+    jrubyDir = join(suitesDir, 'jruby')
+    if os.path.isdir(jrubyDir):
+        mx.run(['git', 'checkout', rubyBranch], cwd=jrubyDir)
+        mx.run(['git', 'pull'], cwd=jrubyDir)
+    else:
+        mx.run(['git', 'clone', rubyUrl], cwd=suitesDir)
```

Comment:
I think Travis has some kind of caching or direct link to GitHub, because this clone only takes a second or two anyway.